projects/igniteui-angular/src/lib/data-operations/filtering-expressions-tree.ts

```diff
@@ -19,7 +21,14 @@ export declare interface IFilteringExpressionsTree extends IBaseEventArgs, IExpr
     /* alternateName: treeType */
     type?: FilteringExpressionsTreeType;
 
+    /**
+     * @deprecated
```

Comment:
Revert this. The  change has been done separate in https://github.com/IgniteUI/igniteui-angular/pull/14815 .

Reply:
Done